source/features/add-swap-branches-on-compare.js

```diff
@@ -0,0 +1,17 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/utils';
+import {getRepoPath, getRepoURL} from '../libs/page-detect';
+
+export default async () => {
+	const references = getRepoPath().replace('compare/', '').split('...').reverse();
+
+	// Compares against the "base" branch if the URL only has one reference
+	if (references.length === 1) {
+		references.unshift(select('.branch span').textContent);
+	}
+
+	const icon = select('.octicon-arrow-left');
+	icon.parentNode.attributes['aria-label'].value += '. Click to swap.';
```

Comment:
Would look better if `Click to swap.` was on a new line.

`'. Click to swap.'` => `'.\nClick to swap.'`